src/ui/windows/TogglDesktop/TogglDesktop/ui/windows/PreferencesWindow.xaml

```diff
@@ -408,6 +408,15 @@
                                          Name="proxyPasswordBox" x:FieldModifier="private"/>
                         </Grid>
                     </StackPanel>
+

```

Comment:
Is this actually enough to display the option in the settings? I didn't see it in my own build nor in the build from Artifacts.